app/(root)/layout.tsx

```diff
@@ -1,7 +1,15 @@
+import LeftSidebar from "@/components/shared/layout/LeftSidebar";
+// import RightSidebar from "@/components/shared/layout/RightSidebar";
 import React from "react";
 
-const RootLayout = () => {
-  return <div>RootLayout</div>;
+const MainLayout = ({ children }: { children: React.ReactNode }) => {
+  return (
+    <main className="bg-black-900">
+      <LeftSidebar />
+      {children}
+      {/* <RightSidebar /> */}
```

Comment:
When pushing code into main we don't want to litter it with commented out components like this. Let's keep our codebase clean. If we need the RightSidebar, keep it. If we don't remove the comment. Main isn't for testing.

Reply:
this isn't the main branch, its the feature_sidebar branch , no ?

Reply:
This is the feature_sidebar branch, that you have made a pull request to have merged INTO main. So any code you write in this PR is meant to go into main. If we merge this code- then:

```
{/* <RightSidebar /> */}
```
Will be in your main branch.

Reply:
yeah I understood that, I thought because I was working on this and not preparing to merge it into main, just intermittently pushing code, I knew I wasn't going to be pushing it into main, I thought the point of opening these pull requests was to get intermitent feedback.  I had commented that out because i hadn't done the styling correctly for it yet and the sidebar was not aligned yet, so I had put my temporary home code into a route named /home so I could see what its doing, while adding that route to the matcher in the middleware so I could bypass the auth while developing.  So if you want to explain to me exactly how this process should be working, when is and when isnt an appropriate time to make a PR (I had made it because you asked me to, not because I was ready to merge it into main, so I'm sure I'm misunderstanding something).